plottable.css

```diff
@@ -44,6 +44,15 @@ svg.plottable {
   pointer-events: visibleFill;
 }
 
+/* Root-level html components are relative, all others are absolute */
+div.plottable {
```

Comment:
@adidahiya could you provide some reading on that? I looked into this before but only found ~3 year old resources, along with other resources telling me not to pay attention to those old things since browsers have been improving perf